lib/libspl/include/sys/time.h

```diff
@@ -58,6 +58,12 @@
 #define	NSEC2MSEC(n)    ((n) / (NANOSEC / MILLISEC))
 #endif
 
+#ifndef HAVE_HR_TYPES
```

Comment:
it was defined in my patch at this position:
https://github.com/zfsonlinux/zfs/pull/2604/files#diff-ac7f7e9da5376d902f6b4f3de8695db6R62